.github/integration/sda-s3-integration.yml

```diff
@@ -317,13 +317,78 @@ services:
       - ./sda/config.yaml:/config.yaml
       - shared:/shared
 
+  cega-nss:
+    container_name: cega-nss
+    depends_on:
+      credentials:
+        condition: service_completed_successfully
+    command:
+      [
+        "python",
+        "/cega/users.py",
+        "0.0.0.0",
+        "8443",
+        "/shared/users.json"
+      ]
+    environment:
+      - CEGA_USERS_PASSWORD=test
+      - CEGA_USERS_USER=test
+    image: "egarchive/lega-base:release.v0.2.0"
+    ports:
+      - "8443:8443"
+    volumes:
+      - ./sda/users.py:/cega/users.py
+      - shared:/shared
+
+  auth-cega:
+    command: [ sda-auth ]
+    container_name: auth-cega
+    depends_on:
+      cega-nss:
+        condition: service_started
+    environment:
+      - AUTH_RESIGNJWT=true
+      - AUTH_CEGA_ID=test
+      - AUTH_CEGA_SECRET=test
+    image: ghcr.io/neicnordic/sensitive-data-archive:PR${PR_NUMBER}
+    ports:
+      - "8888:8080"
+    restart: always
+    volumes:
+      - ./sda/config.yaml:/config.yaml
+      - shared:/shared
+
+  auth-oidc:
+    command: [ sda-auth ]
+    container_name: auth-oidc
+    depends_on:
+      oidc:
```

Comment:
Using the python mock oidc limits using this compose as a dev environment since the python oidc lacks an authorization endpoint and auth will not work.